tests/conftest.py

```diff
@@ -0,0 +1,270 @@
+                        "type": "independent",
+                    },
+                    "claimstore": {
+                        "description": "ClaimStore",
+                        "maintainers": ["jbenito3", "tiborsimko"],
+                        "state": "alpha",
+                        "travis": {"pypideploy": False},
+                        "type": "service",
+                    },
+                    "counter-robots": {
+                        "description": "Library for COUNTER-compliant detection of machines and robots.",
+                        "maintainers": ["slint", "ChiaraBi"],
+                        "state": "stable",
+                        "type": "independent",
+                    },
+                    "datacite": {
+                        "description": "Python API wrapper for the DataCite Metadata Store API.",
+                        "maintainers": ["lnielsen"],
+                        "state": "stable",
+                        "type": "independent",
+                    },
+                    "doschema": {
+                        "description": "Utilities to work with JSON Schemas.",
+                        "maintainers": ["dinosk"],
+                        "state": "alpha",
+                        "type": "independent",
+                    },
+                    "flask-breadcrumbs": {
+                        "description": "Flask-Breadcrumbs is a Flask extension that adds support for generating\nsite breadcrumb navigation.",
+                        "maintainers": ["slint", "drjova"],
+                        "state": "stable",
+                        "type": "independent",
+                    },
+                    "invenio": {
+                        "description": "Invenio digital library framework",
+                        "branches": [
+                            "master",
+                            "legacy",
+                            "maint-3.1",
+                            "maint-3.0",
+                            "maint-2.1",
+                            "maint-2.0",
+                            "maint-1.2",
+                            "maint-1.1",
+                            "maint-1.0",
+                            "maint-0.99",
+                        ],
+                        "maintainers": ["egabancho", "lnielsen", "tiborsimko"],
+                        "state": "beta",
+                        "type": "core",
+                        "has_wiki": True,
+                    },
+                },
+            }
+        }
+    }
+    d = tmpdir_factory.mktemp("tests").join("repositories.yml")
+    with open(d, "w") as outfile:
+        yaml.dump(mock, outfile, default_flow_style=False)
+    return str(d)
+
+
+@pytest.fixture
+def config(mock_info):
+    """Configuration testing fixture."""
+    from autobot.config import Config
+
+    return Config(
+        owner="inveniosoftware", github_token="some token", info_path=mock_info
+    )
+
+
+# @pytest.fixture
```

Comment:
This can be removed for now (let's see later how GitHub mocking is going to work)